backend/src/routes/db_lock/index.js

```diff
@@ -0,0 +1,33 @@
+const controller = require("@controllers/db_lock");
+const what = "db_lock";
+
+const routes = [
+  {
+    method: "POST",
+    url: `/${what}`,
+    // schema: validators.addOne,
+    handler: controller.addLockByParams,
+  },
+  {
+    method: "GET",
+    url: `/${what}`,
+    // schema: validators.getOne,
+    handler: controller.getLockByParams,
+  },
+  {
+    method: "DELETE",
+    url: `/${what}/:id`,
+    // schema: validators.addOne,
```

Comment:
Are the validators not working still?

Reply:
They will be added to another ticket because they are not working properly.